tools/mason/MasonExample.chpl

```diff
@@ -289,7 +289,9 @@ private proc runExampleBinary(projectHome: string, exampleName: string,
 
   const exampleResult = runWithStatus(command, true);
   if exampleResult != 0 {
-    throw new owned MasonError("Mason failed to find and run compiled example: " + exampleName + ".chpl");
+    throw new owned MasonError("Example has not been compiled: " + exampleName + ".chpl\n
```

Comment:
@klakho0400 - you will discover that this does not compile when you test this (try `make mason`). Multi-line strings require a backslash for continuation, e.g.

```
"Example has not been compiled: " + exampleName + ".chpl\n \
```

Reply:
> @klakho0400 - you will discover that this does not compile when you test this (try `make mason`). Multi-line strings require a backslash for continuation, e.g.
> 
> ```
> "Example has not been compiled: " + exampleName + ".chpl\n \
> ```

@ben-albrecht Thanks for the step by step instruction I was able to test it and i added the backslashes after seeing that it did not pass the test but after adding the backslashes I am
still getting 1 failure [Error matching program output for mason/mason-test-exit/exitCodeTest]
and I warning [Warning: SKIPIF error.] How do I resolve them.

Reply:
>  [Warning: SKIPIF error.]

This was an issue on master that is now fixed (you rebase or pull from upstream master to get those changes), or just not worry about this warning).

> Error matching program output for mason/mason-test-exit/exitCodeTest

This test may be impacted by your changes. You need to determine why it is failing and if the expected output (`.good` file) needs to be updated. See the testing docs for more info.